npm-shrinkwrap.json

```diff
@@ -1,7756 +1,8174 @@
 {
   "name": "fxa-oauth-server",
   "version": "1.106.0",
+  "lockfileVersion": 1,
```

Comment:
I had an idea to parse and check to make sure new commits have `"lockfileVersion": 1,` (this will check to make sure the new wrap was created with npm5) but we might not need this